src/routers/routers.ts

```diff
@@ -0,0 +1,44 @@
+
+import { createRouter, createWebHistory } from 'vue-router';
+import App from '../App.vue';
+import HomePage from '../components/HomePage/HomePage.vue';
+import Teste from '../components/Teste.vue';
+import FormUser from '../components/FormUser/FormUser.vue';
+
+export interface RouteSchema {
+    name: string;
+    isDefault: boolean;
+    path: string;
+    component: any;
+  }  
+ 
+  export const routes: RouteSchema[] = [
```

Comment:
Also check the indentation please.
```suggestion
  const routes: RouteSchema[] = [
```